compiler/rustc_codegen_llvm/src/back/lto.rs

```diff
@@ -667,6 +668,12 @@ pub(crate) fn run_pass_manager(
         write::llvm_optimize(cgcx, dcx, module, None, config, opt_level, opt_stage, stage)?;
     }
 
+    if cfg!(llvm_enzyme) && enable_gpu && !thin {
```

Comment:
There is 0 dependency of offload on Enzyme, but since I think I'm support to gate my features, I'll just re-use the ones from Enzyme.

Reply:
I added a span to the error, but emit_error now fails with:

```

thread 'coordinator' panicked at compiler/rustc_codegen_ssa/src/back/write.rs:1908:9:
assertion `left == right` failed
  left: MultiSpan { primary_spans: [Span { lo: BytePos(123), hi: BytePos(148), ctxt: #0 }], span_labels: [] }
 right: MultiSpan { primary_spans: [], span_labels: [] }
stack backtrace:
```

Do you have a recommendation for how to best emit the error? I had some issues to find the right function to get it to work.